integration/kubernetes/k8s-empty-dirs.bats

```diff
@@ -26,9 +38,27 @@ setup() {
 	kubectl exec $pod_name -- sh -c "$cmd" | grep "/tmp/cache type tmpfs"
 }
 
+@test "Empty dir volume when FSGroup is specified with non-root container" {
+	# This is a reproducer of k8s e2e "[sig-storage] EmptyDir volumes when FSGroup is specified [LinuxOnly] [NodeFeature:FSGroup] new files should be created with FSGroup ownership when container is non-root" test
+	pod_file="${pod_config_dir}/pod-empty-dir-fsgroup.yaml"
+	kubectl create -f "$pod_file"
+
+	cmd="kubectl get pods ${pod_name} | grep Completed"
+	waitForProcess "${wait_time}" "${sleep_time}" "${cmd}"
+
+	pod_logs_file="$(mktemp)"
+	kubectl logs "$pod_name" > "$pod_logs_file"
+	uid=$(cat $pod_logs_file | grep 'owner UID of' | sed 's/.*:\s//')
+	assert_equal "1001" "$uid" "owner UID of file"
+	gid=$(cat $pod_logs_file | grep 'owner GID of' | sed 's/.*:\s//')
+	assert_equal "123" "$gid" "owner GIG of file"
```

Comment:
typo 
also i suppose you forgot to do something with $3 in assert_equal :)

Reply:
@snir911 even after you mention, it took me minutes to realize the typo was in "GIG". I call this "super-typo" :)

And regarding the "$3" parameter to assert_equal, the initial plan was to simple print it. But then I noticed the msg would appear on the stack track in case the assert fails. I agree it is weird and I will re-do it.

Thanks for the review!

Reply:
well, is it actually a typo if you don't print it? dunno, it's like a tree falls in the forest and no one is around to hear it :)

Reply:
@snir911 on the updated version which I just pushed I removed the 3rd argument, so it should be fine now. Btw, I like the tree fall analogy :D